### PR TITLE
Add contextual copy to World page

### PR DESCRIPTION
**Pull request by chao-xian**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Adds some additional text to the World location page, as requested from FCDO.

[Trello](https://trello.com/c/WIdOvZXR/737-update-help-and-services-around-the-world-page)

### Before

Screenshot 2022-01-26 at 5 54 14 pm
Screenshot 2022-01-26 at 5 54 33 pm

### After

Screenshot 2022-01-26 at 5 53 55 pm
Screenshot 2022-01-26 at 5 54 04 pm

### CODE REVIEW

File: app/views/world_locations/index.html.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@
         title: sanitize("Help and services around the&nbsp;world")
       } %>
 
+      <p class="govuk-body">Find out what is help is available from the UK government if you're abroad.</p>
```
Comment (maxgds) [inline app/views/world_locations/index.html.erb:11]: Can these strings go in the locales file? Looks like there are some others on this template that could do with shifting over there too if you've time.

File: db/seeds.rb [hunk 2]
```diff
@@ -49,3 +49,10 @@
     world_location_type_id: 1,
   )
 end
+
+if WorldLocation.where(name: "Test International Delegation").blank?
```
Comment (maxgds) [inline db/seeds.rb:53]: Bit surprised to see this in this PR as it doesn't seem related to the trello request but it looks fine.

**Review by maxgds — CHANGES_REQUESTED** [review]

**Review by maxgds — CHANGES_REQUESTED** [review]
See comment for suggested change re locale files

File: app/views/world_locations/index.html.erb (hunk 1 above)
Comment (chao-xian) [reply app/views/world_locations/index.html.erb:11]: Good point I did think that and forgot!

**Review by maxgds — APPROVED** [review]
